enclave_build/src/constants.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use oci_distribution::manifest;
+
+/// Docker inspect architecture constants
+pub const DOCKER_ARCH_ARM64: &str = "arm64";
```

Comment:
are those constants in use somewhere?

Reply:
No, they were initially in docker.rs and I moved them here, will remove them in next commit.